po/fr.po

```diff
@@ -26,28 +26,27 @@ msgstr "Bazaar"
 
 #: data/io.github.kolunmi.Bazaar.desktop.in:8
 msgid "GTK;System;PackageManager;Discover;Flatpak;Software;Store;"
-msgstr "GTK;Système;GestionnaireDePaquets;Discover;Flatpak;Logiciel;Store;"
+msgstr ""
+"GTK;Système;Gestionnaire de paquets;Découvrir;Flatpak;Logiciel;Magasin;"
```

Comment:
Here Discover does not need to be translated since it's Discover as the KDE app store